src/test/java/com/microsoft/azure/servicebus/jms/aad/QueueTest.java

```diff
@@ -0,0 +1,181 @@
+package com.microsoft.azure.servicebus.jms.aad;
+
+import org.junit.Test;
+
+public class QueueTest {
+	TestInitialization initialization;
+	
+	@Test
+	public void e2EQueueTestWithSecretCredential()throws Exception {
+		System.out.println("\nTest send and received queue on the app using SecretCredential");
+		System.out.println("---------------------------------------------------------");
+		this.initialization = new TestInitialization("endToEndSecretCredentialQueue");
+		
+		this.initialization.startAad("Queue", this.initialization.GetSecretCredential());
+		this.initialization.entitiy = initialization.jmsContext.createQueue(initialization.entityName);
+        System.out.println("Queue was created.......");
+        
+		this.initialization.consumer = initialization.jmsContext.createConsumer(initialization.entitiy);
```

Comment:
>entitiy [](http://example.com/codeflow?start=89&length=7)

nit: spell.
Please see if you can install a spell check plugin for eclipse and run that so we can remove other places that may have a spelling correction.